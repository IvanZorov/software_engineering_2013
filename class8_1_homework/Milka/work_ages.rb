require 'csv'

array=Array.new()
age_1=ARGV[2]
age_2=ARGV[3]
CSV.foreach(ARGV[0]) do |row|
	if row[1] >= age_1 && row[1] <= age_2 && row[2] == ARGV[1]
	array << [row[0],row[3]]
	end
end
array.sort!{|a,b| b[1].to_f<=>a[1].to_f}
CSV.open("work_ages_result.csv","w") do|csv|
array.each do |element|
csv << element

end
end
