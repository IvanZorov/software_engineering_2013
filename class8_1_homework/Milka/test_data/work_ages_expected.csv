Magaret Caesar,60590.20
Boris Delafuente,6400.50
Glenda Endo,5499.24
Charita Woomer,3630.53
Samella Tufts,2410.00
Kellie Paulin,2400.00
Robbi Liang,1674.35
Chastity Budd,1585.00
